src/components/grid-aware/HomePageLargeParagraph/HomePageParagraph.module.css

```diff
@@ -0,0 +1,35 @@
+.bleedWrapper {
+  justify-items: center;
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+  padding-bottom: 120px;
+  padding-top: 120px;
+}
+
+.bleedMainContent {
+  grid-column: 2;
+  grid-row: 1;
+  /* Override grid item default of auto so that item contents cannot stretch the grid itself */
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    grid-template-columns: 40px 1fr 40px;
+  }
+}
+
+.title {
```

Comment:
This should also be center-aligned to match the design. It's hard to notice in your example, since the example is wide enough to almost exactly fit the width, but if you delete some of the text in the example, you can see that it is left-aligned.

Also, you can set the `text-align` on a higher up element, such as the `<div>` that you have that currently lacks a class name, since it'll apply to all descendants.

Reply:
Instead, can I delete that empty div and set text-align to the bleedMainContent property?

Reply:
Yeah, I think you can do that, which I see you've already done.